auto_landing/cfg/camera_params.yaml

```diff
@@ -1,9 +1,17 @@
-height: 480
-width: 640
-distortion_model: "plumb_bob"
-distortion_matrix: [0.0, 0.0, 0.0, 0.0, 0.0]
-camera_matrix: [320.25492609007654, 0.0, 320.5, 0.0, 320.25492609007654, 240.5, 0.0, 0.0, 1.0]
-rectification_matrix: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0]
-projection_matrix: [320.25492609007654, 0.0, 320.5, -0.0, 0.0, 320.25492609007654, 240.5, 0.0, 0.0, 0.0, 1.0, 0.0]
-camera_to_quad_matrix: [-1, 0, 0, 0, -1, 0, 0, 0, -1]
-camera_translation: [0.20, 0.0, -0.02]
+##Camera Parameters
+distortion_matrix:                                                                      #1x5 matrix of double values used for undistorting distorted images
+  rows: 1
+  cols: 5
+  data: [0.0, 0.0, 0.0, 0.0, 0.0]
+camera_matrix:                                                                          #1x5 matrix of double values used for image preprocessing 
+  rows: 1
+  cols: 5
+  data: [320.25492609007654, 0.0, 320.5, 0.0, 320.25492609007654, 240.5, 0.0, 0.0, 1.0]
+camera_to_quad_matrix:                                                                  #3x3 matrix of int values used for estimating the postion of the platform
```

Comment:
This is a 3x3 rotation matrix (not necessarily of int values) that is used to rotate a vector from the camera to the quad frame.